dashboard/app/models/authentication_option.rb

```diff
@@ -26,12 +26,17 @@ class AuthenticationOption < ApplicationRecord
   belongs_to :user
 
   # These are duplicated from the user model, until we're ready to cut over and remove them from there
-  before_save :normalize_email, :hash_email, :fix_by_user_type
+  before_save :normalize_email, :hash_email, :remove_student_cleartext_email,
+    :fill_authentication_id
```

Comment:
I renamed a couple of these hooks - `fix_by_<property>` seemed poorly descriptive, a legacy thing we can leave behind now.